runtime/service.go

```diff
@@ -670,6 +672,20 @@ func (s *service) startVM(ctx context.Context,
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to build VM options")
 	}
+
+	// Stub out drives that will be eventually replaced with actual drives.
+	tempDir, err := ioutil.TempDir("", "stub-drives")
```

Comment:
We'll probably want to add a defer here that cleans up the temp dir in case we encounter an error.

That also brings up the point that, even if everything is successful, we should remove these directories when the VM shuts down